content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -118,7 +118,13 @@ Containers:
    state of readiness before the initial delay is `Failure`. If a Container does
    not provide a readiness probe, the default state is `Success`.
 
-### When should you use liveness or readiness probes?
+* `startupProbe`: Indicates whether the application within the Container is started.
```

Comment:
We usually write container in all-lower-case:
```suggestion
* `startupProbe`: Indicates whether the application within the container is started.
```
as it's not an object in Kubernetes (unlike say Pod).

The page as it stands uses both forms (“container” and “Container”); let's leave room to fix the other uses in a separate PR &mdash; if someone wants to / has time to.

Reply:
So I should update all `Containers` into `containers` in these 2 pages, and then do a global fix at a later time?
Looks like to me almost all other pages are referencing containers as `Containers` as they do exist in the [API](https://github.com/kubernetes/kubernetes/blob/fb41b7a801538c6c8c9385dc35bf1eaae92f06ab/pkg/apis/core/types.go#L1983).

Reply:
I guess it's fine to leave it as is.